python/tank/bootstrap/configuration_writer.py

```diff
@@ -284,15 +284,16 @@ def create_tank_command(self, executable=sys.executable, prefix=sys.prefix):
 
         config_root_path = self._path.current_os
 
+        # Write out missing files.
         for platform in executables:
             sg_config_location = os.path.join(
                 config_root_path,
                 "config",
                 "core",
                 "interpreter_%s.cfg" % platform
             )
-            # clean out any existing files
-            filesystem.safe_delete_file(sg_config_location)
+            if os.path.exists(sg_config_location):
```

Comment:
We don't have such documentation in tk-core. Maybe something for the admin guide?